src/lib/components/workflow/input-and-results-payload.svelte

```diff
@@ -2,31 +2,58 @@
   import CodeBlock from '$lib/holocene/code-block.svelte';
   import { translate } from '$lib/i18n/translate';
   import { minimizeEventView } from '$lib/stores/event-view';
+  import type { Payload } from '$lib/types';
+  import type { PotentiallyDecodable } from '$lib/utilities/decode-payload';
   import {
     parseWithBigInt,
     stringifyWithBigInt,
   } from '$lib/utilities/parse-with-big-int';
 
-  import {
-    getPayloads,
-    parseContent,
-    parsePayloads,
-  } from './input-and-result-payloads';
   import PayloadDecoder from '../event/payload-decoder.svelte';
 
-  export let title: string;
-  export let content: string;
-  export let isRunning: boolean;
+  type Props = {
+    title: string;
+    content: string;
+    isRunning: boolean;
+  };
+
+  let { title, content = '', isRunning }: Props = $props();
```

Comment:
Could we just default `isRunning` to false here and remove it from `schedule-input.svelte`?

```suggestion
  let { title, content = '', isRunning = false }: Props = $props();
```